app/config/hr16/drush.make.tmpl

```diff
@@ -85,9 +85,9 @@ projects[webform_civicrm][subdir] = contrib
 projects[webform_civicrm][version] = "4.16"
 
 projects[views][subdir] = contrib
-projects[views][version] = 3.11
+projects[views][version] = 3.14
 projects[views][patch][] = "https://www.drupal.org/files/issues/views-exposed_forms_ajax_support-1183418-73.patch"
-projects[views][patch][] = "https://www.drupal.org/files/issues/views-exposed-ajax-not-working-2425099-52.patch"
+;projects[views][patch][] = "https://www.drupal.org/files/issues/views-exposed-ajax-not-working-2425099-52.patch"
```

Comment:
I think it is better to remove the line rather than commenting it out .. if we want it back later we can just check git log .